### PR TITLE
Use business_hours to calculate duration in swimlane

### PR DESCRIPTION
**Pull request by jcleary**
Connects #32

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -33,6 +33,7 @@ gem 'cancancan', '~> 2.0.0'
 gem 'github_webhook', '~> 1.1'
 gem 'clockwork'
 gem 'sidekiq'
+gem 'business_time'
```
Comment (hass-laughingman) [inline Gemfile]: https://gph.is/1J3jtWo

Comment (hass-laughingman) [inline Gemfile]: giphy

Comment (hass-laughingman) [reply Gemfile]: i kinda wanted to spam this giphy anywhere it says `business_time`

Comment (jcleary) [reply Gemfile]: Why do you think I picked this gem?

**Pull request merged**